rapidstream-demo/sextans/design/run_tapa.sh

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2024 RapidStream Design Automation, Inc. and contributors.
+# All rights reserved. The contributor(s) of this file has/have agreed to the
+# RapidStream Contributor License Agreement.
+
+WORK_DIR=generated
+
+tapac \
```

Comment:
tapac does not exist in the current tapa version. Please check if this file is still in use and if your tapa is the most current version?

Reply:
just updated tapa today, using `tapa compile` now instead of tapac